modules/sysrepo-plugind.yang

```diff
@@ -0,0 +1,19 @@
+module sysrepo-plugind {
+    namespace "urn:sysrepo:plugind";
+    prefix srpd;
+
+    revision 2020-12-10 {
+        description "Initial revision.";
+    }
+
+    container sysrepo-plugind {
+        container plugin-order {
+            description "The order in which to run plugins from the sysrepo-plugind.";
+            leaf-list plugin {
+                description "Plugin name.";
```

Comment:
Now it would make sense to include details about what this exactly means. Specifically, that it should be the plugin file name without the extension, if any.